nbdime/prettyprint.py

```diff
@@ -42,29 +43,63 @@
 # TODO: Use this for line wrapping some places?
 MAXWIDTH = 78
 
+git_diff_print_cmd = 'git diff --no-index --color-words before after'
+diff_print_cmd = 'diff before after'
+git_mergefile_print_cmd = 'git merge-file -p local base remote'
+diff3_print_cmd = 'diff3 -m local base remote'
+
 if use_colors:
     import colorama
     RED = colorama.Fore.RED
     GREEN = colorama.Fore.GREEN
     BLUE = colorama.Fore.BLUE
     YELLOW = colorama.Fore.YELLOW
     RESET = colorama.Style.RESET_ALL
-    _git_diff_print_cmd = 'git diff --no-index --color-words'
+
 else:
     RED = ''
     GREEN = ''
     BLUE = ''
     YELLOW = ''
     RESET = ''
-    _git_diff_print_cmd = 'git diff --no-index'
+    git_diff_print_cmd = git_diff_print_cmd.replace(" --color-words ", "")
+
 
 KEEP     = '{color}   '.format(color='')
 REMOVE   = '{color}-  '.format(color=RED)
 ADD      = '{color}+  '.format(color=GREEN)
 INFO     = '{color}## '.format(color=BLUE)
 DIFF_ENTRY_END = '\n'
 
-def _external_diff_render(cmd, a, b):
+
+def external_merge_render(cmd, b, l, r):
+    try:
+        if isinstance(l, bytes):
+            l = l.decode("utf8")
+        if isinstance(b, bytes):
+            b = b.decode("utf8")
+        if isinstance(r, bytes):
+            r = r.decode("utf8")
+        td = tempfile.mkdtemp()
```

Comment:
Maybe move preceding lines outside `try..finally`? Avoids having undefined `td` in `finally` block.